### PR TITLE
feat: filter collectibles

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-11340](https://app.clickup.com/t/2645698/DEV-11340)

### Description

Add filters to collectibles view.

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11340 Add search and filters in collection page](https://app.clickup.com/t/2645698/DEV-11340)

**Review by doubleppereira — APPROVED** [review]
LGTM just a few comments

File: utils/validateLspMetadata.ts [hunk 1]
```diff
@@ -180,7 +180,16 @@ export const validateLinks = (links: unknown): LinkMetadata[] => {
   return (
     (Array.isArray(links) &&
       links?.filter(link => {
-        return link?.title && link?.url
+        const isValidURL = (url: string): boolean => {
```
[the remark this answers is not in the document]
Comment (dzbo) [reply utils/validateLspMetadata.ts:183]: Makes sense 👍🏻

File: components/CollectionView.vue [hunk 2]
```diff
@@ -8,32 +12,129 @@ const { isRpc } = useDataProvider()
...
+    data.value = []
+  }
+
+  // if we reached the end of the collection
+  if (total.value !== null && offset.value >= total.value) {
+    return
+  }
+
+  isLoading.value = true
+  await sleep(250)
```
[the remark this answers is not in the document]
Comment (dzbo) [reply components/CollectionView.vue:62]: Yes this is intended delay (works as debounce effect) so next page doesn't load right away after another.

**Pull request merged**